model/src/main/java/oracle/kubernetes/weblogic/domain/model/Istio.java

```diff
@@ -11,25 +11,47 @@
 
 public class Istio {
 
-  @Description("True, if this domain is deployed under an Istio service mesh. Defaults to true. Not required.")
-  private Boolean istioEnabled = true;
+  @Description(
+      "True, if this domain is deployed under an Istio service mesh. Defaults to true. Not required.")
```

Comment:
istio should default to false

Reply:
fixed

Reply:
My intention was that istioEnabled here defaults to true...  The idea was that Istio is disabled by default, but enabled by default once the "istio: {}" element is included in domain.yaml.  So...

1. Default
Istio enablement is disabled

2. Istio element present
spec:
  istio: {}
Istio enablement is enabled

3. Istio element present, but disabled
spec:
  istio:
    istioEnabled = false
    ... other settings preserved
Istio enablement is disabled.

Also, I think I would rename "istioEnabled" to "enabled"

Reply:
Spacing got lost after I posted the comment...  If it's confusing, let me know.